pkgs/by-name/ai/airwin2rack/package.nix

```
@@ -0,0 +1,215 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  pkgs,
+  lib,
+  makeDesktopItem,
+  copyDesktopItems,
+  srcOnly,
+  cmake,
+  pkg-config,
+  alsa-lib,
+  xorg,
+  freetype,
+  libGLU,
+  libjack2,
+  juce,
+  webkitgtk_4_0,
+  libsysprof-capture,
+  pcre2,
+  util-linux,
+  libselinux,
+  libsepol,
+  libthai,
+  libxkbcommon,
+  libdatrie,
+  libepoxy,
+  libsoup_2_4,
+  lerc,
+  sqlite,
+  glib,
+  gtk3-x11,
+  curl,
+  enableVCVRack ? false,
+}:
+let
+  clapJuceExtensions = fetchFromGitHub {
+    owner = "free-audio";
+    repo = "clap-juce-extensions";
+    rev = "4f33b4930b6af806018c009f0f24b3a50808af99";
+    hash = "sha256-M+T7ll3Ap6VIP5ub+kfEKwT2RW2IxxY4wUPRQKFIotk=";
+    fetchSubmodules = true;
+  };
+
+  vcvRackSdk = srcOnly pkgs.vcv-rack;
+  pname = "airwin2rack";
+  version = "2.13.0";
+in
+stdenv.mkDerivation {
+  inherit pname;
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "baconpaul";
+    repo = "airwin2rack";
+    rev = "db56d13f853831ab94a5e1713282e4e518f50d5c";
+    hash = "sha256-utqDmQgnYUtUv0E0xhO5rGx+9RXTAn8kKhTzkyXjcbE=";
+    fetchSubmodules = true;
+  };
+
+  desktopItems = [
+    (makeDesktopItem {
+      type = "Application";
+      name = "Airwin2rack";
+      desktopName = "Airwindows Consolidated";
+      comment = "Various Airwindows Plugins Consolidated (Standalone)";
+      exec = "Airwindows Consolidated";
+      categories = [
+        "Audio"
+        "AudioVideo"
+      ];
+    })
+  ];
+
+  strictDeps = true;
+
+  nativeBuildInputs =
+    [
+      cmake
+      pkg-config
+      copyDesktopItems
+    ]
+    ++ lib.optionals enableVCVRack [
+      pkgs.jq
+      pkgs.zstd
+    ];
+
+  buildInputs =
+    [
+      alsa-lib
+      xorg.libX11
+      xorg.libXcomposite
+      xorg.libXcursor
+      xorg.libXext
+      xorg.libXinerama
+      xorg.libXrandr
+      xorg.libXrender
+      xorg.libXtst
+      xorg.libXdmcp
+      libGLU
+      libjack2
+      freetype
+      webkitgtk_4_0
+      glib
+      gtk3-x11
+      curl
+      libsysprof-capture
+      pcre2
+      util-linux
+      libselinux
+      libsepol
+      libthai
+      libxkbcommon
+      libdatrie
+      libepoxy
+      libsoup_2_4
+      lerc
+      sqlite
+    ]
+    ++ lib.optionals enableVCVRack [
+      pkgs.vcv-rack
+      pkgs.jansson
+      pkgs.glew
+      pkgs.glfw
+      pkgs.libarchive
+      pkgs.speexdsp
+      pkgs.libpulseaudio
+      pkgs.libsamplerate
+      pkgs.rtmidi
+      pkgs.zstd
+    ];
+
+  cmakeFlags =
+    [
+      (lib.cmakeBool "BUILD_JUCE_PLUGIN" true)
+      (lib.cmakeBool "USE_JUCE_PROGRAMS" true)
+    ]
+    ++ lib.optionals enableVCVRack [
+      (lib.cmakeBool "BUILD_RACK_PLUGIN" true)
+      "-DRACK_SDK_DIR=${vcvRackSdk}"
```

FYI, this should work as well
```suggestion
      (lib.cmakeFeature "RACK_SDK_DIR" "${vcvRackSdk}")
```